Prefabs/UI/Materials/TooltipBackground.mat

```diff
@@ -7,18 +7,18 @@ Material:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
   m_Name: TooltipBackground
-  m_Shader: {fileID: 4800000, guid: 68eb1b178e2b7f74eb314a539c948954, type: 3}
-  m_ShaderKeywords: _ALPHAPREMULTIPLY_ON _EMISSION _METALLICGLOSSMAP
+  m_Shader: {fileID: 46, guid: 0000000000000000f000000000000000, type: 0}
+  m_ShaderKeywords: _EMISSION
   m_LightmapFlags: 1
-  m_CustomRenderQueue: -1
+  m_CustomRenderQueue: 9002
   stringTagMap: {}
   m_SavedProperties:
     serializedVersion: 2
     m_TexEnvs:
     - first:
         name: _BumpMap
       second:
-        m_Texture: {fileID: 2800000, guid: 06dc8da358c6ebb4dad67b286b513f95, type: 3}
+        m_Texture: {fileID: 0}
```

Comment:
It is deliberate.  The texture had to be reassigned to the albedo slot after the shader swap to the standard shader.